.github/workflows/oneshot_candidate_release.yml

```diff
@@ -0,0 +1,56 @@
+name: Schedule candidate release
+
+on:
+  workflow_dispatch:
```

Comment:
This should use a different name ("Create manual candidate release" or "Oneshot candidate release" maybe)

Reply:
sounds good. will do.

Reply:
done